runtime/recipe/slot-connection.js

```diff
@@ -119,12 +119,6 @@ class SlotConnection {
       }
       return false;
     }
-    if (this.slotSpec.isRequired && this.targetSlot.sourceConnection == undefined) {
-      if (options) {
-        options.details = 'missing target-slot\'s source-connection of required connection';
```

Comment:
is this no longer possible? Or are we losing debug messages?

Reply:
this was never correct (if the target slot is "remote" slot, ie not part of this recipe, then this.targetSlot.sourceConnection is undefined and it's ok).
but it was never affecting us, because we never use required slot :(